synapseAnnotations/data/cancer.json

```diff
@@ -197,6 +197,11 @@
         "description": "A malignant, usually aggressive tumor composed of atypical, neoplastic melanocytes.",
         "source": "https://www.ebi.ac.uk/ols/ontologies/mp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C3224"
       },
+      {
+        "value": "Glioma",
```

Comment:
I wish there were some way to nest `"Low Grade Glioma"` and `"Glioblastoma"` under `"Glioma"` 😕 but as there isn't this term is ok with me.

Reply:
Yes, agreed.

Reply:
A possible solution to this would be to start using oncotree codes (where low grade glioma would be LGGNOS, glioma would be GNOS, and glioblastoma multiforme would be GBM 
(where NOS means "not otherwise specified").
more here:
http://oncotree.mskcc.org/#/home